Signal/src/view controllers/CallViewController.swift

```diff
@@ -318,7 +322,12 @@ class CallViewController: UIViewController, CallObserver, CallServiceObserver, R
                     spacer.autoPinEdge(.right, to:.left, of:subview)
                     spacer.setContentHuggingHorizontalLow()
                     spacer.autoVCenterInSuperview()
-                    if lastSpacer != nil {
+
+                    if subviews.count == 2 {
```

Comment:
This won't handle the case where there's more than two buttons in a row. You could modify 

```if subviews.count == 2 ```

to read:

```if row.subviews.count == 2 ```

On the other hand, it might just be better to assert in the `else...if` clause since this would be would too much space for a 3 or 4 button row.